component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/elk/ELKMetricEventBuilder.java

```diff
@@ -85,8 +102,8 @@ private void setUserAgentProperties(String userAgentHeader) {
     private void copyDefaultPropertiesToRootLevel(Map<String, String> properties) {
         String apiContext = properties.remove(Constants.API_CONTEXT);
```

Comment:
If we are not using context and username remove the declaration

Reply:
Ok.